src/encode/mod.rs

```diff
@@ -89,6 +93,11 @@ pub enum Color {
     Cyan,
     White,
 }
+impl Default for Color {
+    fn default() -> Self {
+        Color::Black
```

Comment:
Should it be "None" to match either black or white depending on shell color mode ?